internal/service-manager/types/service_instance.go

```diff
@@ -2,6 +2,9 @@ package types
 
 import "encoding/json"
 
+// Refs
+// https://github.com/SAP/sap-btp-service-operator/blob/main/client/sm/types/service_instance.go
+// https://github.com/Peripli/service-manager/blob/master/pkg/types/service_instance.go
```

Comment:
Below NumItems is not added.

Reply:
This is a model for the Service Manager. I guess that `NumItems` was applicable just in VM structs (View Model), maybe, to show count in the UI.